# Translations template for vigilo-vigiconf.
# Copyright (C) 2014 ORGANIZATION
# This file is distributed under the same license as the vigilo-vigiconf
# project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2014.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: vigilo-vigiconf 3.5\n"
"Report-Msgid-Bugs-To: contact@projet-vigilo.org\n"
"POT-Creation-Date: 2014-10-09 12:48+0200\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 0.9.4\n"

#: src/vigilo/vigiconf/commandline.py:73
#, python-format
msgid "%(option)s has been deprecated. Please use --stop-after=%(value)s instead."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:103
#, python-format
msgid "ERROR: %s"
msgstr ""

#: src/vigilo/vigiconf/commandline.py:106
msgid "No server to manage."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:151
msgid "Available host classes:"
msgstr ""

#: src/vigilo/vigiconf/commandline.py:161
#, python-format
msgid "No such host class '%s'"
msgstr ""

#: src/vigilo/vigiconf/commandline.py:170
#, python-format
msgid "Tests for host class '%s':"
msgstr ""

#: src/vigilo/vigiconf/commandline.py:210
msgid "Debug mode."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:212
msgid "Don't try to switch to the correct user."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:217
msgid "Vigilo configuration manager"
msgstr ""

#: src/vigilo/vigiconf/commandline.py:218
msgid "Subcommands"
msgstr ""

#: src/vigilo/vigiconf/commandline.py:224
msgid "Prints a summary of the current configuration."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:227
msgid "Supervision servers to query, all of them if not specified."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:234
msgid "Lists tests available for certain host classes."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:236
msgid "Host classes to query."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:280
msgid "Deploys the configuration on each server if the configuration has changed."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:286
msgid ""
"[DEPRECATED] Stop the process after the file generation, before pushing the "
"files."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:292
msgid ""
"[DEPRECATED] Stop after pushing the files to the remote servers, before "
"services are restarted."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:297
msgid "OPERATION"
msgstr ""

#: src/vigilo/vigiconf/commandline.py:298
#, python-format
msgid "Stop after the given OPERATION (one of '%s')"
msgstr ""

#: src/vigilo/vigiconf/commandline.py:301
msgid "Deploy the given revision"
msgstr ""

#: src/vigilo/vigiconf/commandline.py:306
msgid ""
"Force the given operations. This can be used to return VigiConf to a known "
"good state."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:310
msgid "Simulate only, no copy will actually be made, no commit in the database."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:314
msgid "Disable generators marked as requiring only the database to operate (dbonly)."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:317
msgid "Supervision servers to deploy to, all of them if not specified."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:324
msgid "Discover the services available on a remote server."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:329
msgid "Output file. Default: standard output."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:331
#, python-format
msgid "SNMP community. Default: %(default)s."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:333
#, python-format
msgid "SNMP version. Default: %(default)s."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:335
#, python-format
msgid "Main group. Default: %(default)s."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:337
msgid "Tests to check."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:339
msgid ""
"Hosts or files to scan. The files must be the result of an snmpwalk command "
"on the '.1' OID with the '-OnQe' options."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:347
msgid "Enables or disables a Vigilo server"
msgstr ""

#: src/vigilo/vigiconf/commandline.py:350
msgid "Don't re-deploy the configuration, only set the status."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:353
msgid "New status"
msgstr ""

#: src/vigilo/vigiconf/commandline.py:355
msgid "Server name(s) to enable/disable"
msgstr ""

#: src/vigilo/vigiconf/commandline.py:370
#, python-format
msgid "VigiConf was launched as user 'root'. Switching to user '%s' instead."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:375
#, python-format
msgid "Unable to switch to user '%s'. Aborting."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:400
#, python-format
msgid "VigiConf was not launched as user '%s'. Aborting."
msgstr ""

#: src/vigilo/vigiconf/commandline.py:435 src/vigilo/vigiconf/commandline.py:437
#, python-format
msgid "VigiConf error: %s"
msgstr ""

#: src/vigilo/vigiconf/commandline.py:441
msgid "VigiConf is done."
msgstr ""

#: src/vigilo/vigiconf/conf.py:111
msgid "Loading XML configuration"
msgstr ""

#: src/vigilo/vigiconf/conf.py:125
msgid "Error loading configuration"
msgstr ""

#: src/vigilo/vigiconf/discoverator.py:161
#, python-format
msgid "%s: No such file"
msgstr ""

#: src/vigilo/vigiconf/discoverator.py:166
msgid "Can't find the hostname using SNMPv2-MIB::sysName.0"
msgstr ""

#: src/vigilo/vigiconf/discoverator.py:170
#, python-format
msgid ""
"Invalid HOSTNAME detected in SNMPv2-MIB::sysName.0: \"%s\" (blank character "
"not allowed)"
msgstr ""

#: src/vigilo/vigiconf/discoverator.py:200
msgid "The \"snmpwalk\" command is not installed."
msgstr ""

#: src/vigilo/vigiconf/discoverator.py:204
#, python-format
msgid ""
"SNMP walk command failed with error status %(status)s and message:\n"
"  %(msg)s\n"
"The command was: %(cmd)s"
msgstr ""

#: src/vigilo/vigiconf/applications/connector_metro/generator.py:173
#, python-format
msgid "Unknown RRA template %s"
msgstr ""

#: src/vigilo/vigiconf/applications/vigirrd/generator.py:88
#, python-format
msgid "Some datasources are not on a graph: %s"
msgstr ""

#: src/vigilo/vigiconf/lib/application.py:89
#, python-format
msgid ""
"A timeout occured while executing application %(application)s with action "
"%(action)s on the following servers: %(servers)s."
msgstr ""

#: src/vigilo/vigiconf/lib/application.py:262
#, python-format
msgid "Can't find the validation script for app %(app)s: %(error)s"
msgstr ""

#: src/vigilo/vigiconf/lib/application.py:436
#, python-format
msgid "%(app)s: validation failed for server '%(server)s': %(reason)s"
msgstr ""

#: src/vigilo/vigiconf/lib/application.py:443
#, python-format
msgid "%(app)s : Validation successful for server: %(server)s"
msgstr ""

#: src/vigilo/vigiconf/lib/application.py:465
#, python-format
msgid "%(app)s : Qualification failed on '%(server)s' - REASON: %(reason)s"
msgstr ""

#: src/vigilo/vigiconf/lib/application.py:474
#, python-format
msgid "%(app)s : Qualification successful on server : %(server)s"
msgstr ""

#: src/vigilo/vigiconf/lib/application.py:489
#, python-format
msgid "Starting %(app)s on %(server)s ..."
msgstr ""

#: src/vigilo/vigiconf/lib/application.py:492
#, python-format
msgid "Reloading %(app)s on %(server)s ..."
msgstr ""

#: src/vigilo/vigiconf/lib/application.py:500
#, python-format
msgid "Can't Start %(app)s on %(server)s - REASON %(reason)s"
msgstr ""

#: src/vigilo/vigiconf/lib/application.py:509
#, python-format
msgid "%(app)s started on %(server)s"
msgstr ""

#: src/vigilo/vigiconf/lib/application.py:511
#, python-format
msgid "%(app)s reloaded on %(server)s"
msgstr ""

#: src/vigilo/vigiconf/lib/application.py:524
#, python-format
msgid "Stopping %(app)s on %(server)s ..."
msgstr ""

#: src/vigilo/vigiconf/lib/application.py:534
#, python-format
msgid "Can't Stop %(app)s on %(server)s - REASON %(reason)s"
msgstr ""

#: src/vigilo/vigiconf/lib/application.py:543
#, python-format
msgid "%(app)s stopped on %(server)s"
msgstr ""

#: src/vigilo/vigiconf/lib/application.py:588
msgid "Empty list of servers"
msgstr ""

#: src/vigilo/vigiconf/lib/application.py:662
#, python-format
msgid ""
"Incoherent configuration: application %(app)s has an entry point named "
"%(epname)s in package %(eppkg)s"
msgstr ""

#: src/vigilo/vigiconf/lib/application.py:667
#, python-format
msgid "Application %s is installed but disabled (see conf.d/general/apps.py)"
msgstr ""

#: src/vigilo/vigiconf/lib/application.py:671
#, python-format
msgid "application %s is not unique. Providing modules: %s"
msgstr ""

#: src/vigilo/vigiconf/lib/application.py:682
#, python-format
msgid "Application %s has been added to conf.d/general/apps.py, but is not installed"
msgstr ""

#: src/vigilo/vigiconf/lib/application.py:697
msgid "validation failed, see above for more information."
msgstr ""

#: src/vigilo/vigiconf/lib/application.py:699
msgid "Validation successful"
msgstr ""

#: src/vigilo/vigiconf/lib/application.py:713
msgid "qualification failed. See above for more information."
msgstr ""

#: src/vigilo/vigiconf/lib/application.py:715
msgid "Qualification successful"
msgstr ""

#: src/vigilo/vigiconf/lib/application.py:756
#, python-format
msgid ""
"Next attempt for %(action)s action in %(interval)d second(s) (attempt "
"#%(attempt)d)"
msgstr ""

#: src/vigilo/vigiconf/lib/systemcommand.py:127
#, python-format
msgid "Command \"%(cmd)s\" failed with code %(code)s and message: %(msg)s"
msgstr ""

#: src/vigilo/vigiconf/lib/systemcommand.py:146
#, python-format
msgid "Missing command: %s"
msgstr ""

#: src/vigilo/vigiconf/lib/validator.py:127
msgid "No server configuration to be generated"
msgstr ""

#: src/vigilo/vigiconf/lib/validator.py:130
msgid "No application configuration to be generated"
msgstr ""

#: src/vigilo/vigiconf/lib/validator.py:133
msgid "No host configuration to be generated"
msgstr ""

#: src/vigilo/vigiconf/lib/validator.py:141
#, python-format
msgid ""
"The number of host entries in the database does not match the number of hosts"
" in the configuration. Database: %(db)d, configuration: %(conf)d"
msgstr ""

#: src/vigilo/vigiconf/lib/validator.py:165
#, python-format
msgid ""
"The number of services entries in the database does not match the number of "
"services in the configuration. Database: %(db)d, configuration: %(conf)d"
msgstr ""

#: src/vigilo/vigiconf/lib/validator.py:179
#, python-format
msgid ""
"The number of apps loaded in the database does not match the number of apps "
"to deploy. Database: %(db)d, to deploy: %(deploy)d. Possible cause: one or "
"more app have no Vigilo server to be deployed on."
msgstr ""

#: src/vigilo/vigiconf/lib/validator.py:197
#, python-format
msgid ""
"The number of ventilation entries in the database does not match the number "
"of host and apps. Database: %(db)d, should be: %(conf)d. Possible cause: one "
"or more application have no Vigilo server to be deployed on."
msgstr ""

#: src/vigilo/vigiconf/lib/validator.py:231
#, python-format
msgid ""
"%(nb_gens)d generators exist whereas %(nb_apps)d applications have been "
"configured"
msgstr ""

#: src/vigilo/vigiconf/lib/validator.py:238
#, python-format
msgid "########> Error in %(source)s (object %(object)s): %(error)s"
msgstr ""

#: src/vigilo/vigiconf/lib/validator.py:245
#, python-format
msgid "====> Warning in %(source)s (object %(object)s): %(error)s"
msgstr ""

#: src/vigilo/vigiconf/lib/validator.py:251
#, python-format
msgid "%d errors"
msgstr ""

#: src/vigilo/vigiconf/lib/validator.py:252
#, python-format
msgid "%d warnings"
msgstr ""

#: src/vigilo/vigiconf/lib/validator.py:254
#, python-format
msgid ""
"%(nbGenerators)d generators have configured %(nbHosts)d hosts on "
"%(nbServers)d servers."
msgstr ""

#: src/vigilo/vigiconf/lib/validator.py:256
#, python-format
msgid "%(filesWritten)d files have been generated in %(dirsCreated)d directories"
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/host.py:141
#: src/vigilo/vigiconf/lib/confclasses/hosttemplate.py:180
#, python-format
msgid ""
"The \"%(old_attribute)s\" attribute has been deprecated. Please use "
"\"%(replacement)s\" instead."
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/host.py:156
#, python-format
msgid "In host %(host)s: attribute name %(attribute)s is reserved"
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/host.py:205
#, python-format
msgid ""
"Test \"%(test_name)s\" on \"%(host)s\" needs the following arguments: "
"%(args)s (and only those)"
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/host.py:540
#, python-format
msgid "host '%(host)s': wrong datasource in graph '%(graph)s': %(ds)s"
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/host.py:564
#, python-format
msgid "CDEF name must be ASCII-only: %s"
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/host.py:769
msgid "Cannot override general configuration."
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/host.py:779
msgid "oxe_login and oxe_password cannot be empty"
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/host.py:782
#, python-format
msgid ""
"Value for '%(attr)s' (%(value)s) could not be cast into an integer. Using "
"default value (%(default)d)."
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/host.py:838
#, python-format
msgid "Cannot add tag \"%(tag)s\" to non-existing service \"%(service)s\""
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/host.py:861
#, python-format
msgid ""
"Empty value given for directive %(directive)s with target %(target)s on host "
"%(host)s"
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/host.py:929
#: src/vigilo/vigiconf/lib/confclasses/hosttemplate.py:321
#: src/vigilo/vigiconf/lib/loaders/xmlloader.py:97
#, python-format
msgid "Invalid XML validation schema %(schema)s: %(error)s"
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/host.py:935
#: src/vigilo/vigiconf/lib/confclasses/host.py:955
#: src/vigilo/vigiconf/lib/confclasses/hosttemplate.py:327
#: src/vigilo/vigiconf/lib/confclasses/hosttemplate.py:347
#: src/vigilo/vigiconf/lib/loaders/xmlloader.py:103
#: src/vigilo/vigiconf/lib/loaders/xmlloader.py:123
#, python-format
msgid ""
"Error reading %(file)s, make sure the permissions are set correctly.Message: "
"%(error)s."
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/host.py:952
#: src/vigilo/vigiconf/lib/confclasses/hosttemplate.py:344
#: src/vigilo/vigiconf/lib/loaders/xmlloader.py:120
#, python-format
msgid "XML syntax error in %(file)s: %(error)s"
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/host.py:963
#: src/vigilo/vigiconf/lib/confclasses/hosttemplate.py:355
#: src/vigilo/vigiconf/lib/loaders/xmlloader.py:131
#, python-format
msgid "XML validation failed: %(error)s"
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/host.py:1024
#, python-format
msgid "Invalid ventilation group: %s"
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/host.py:1065
#, python-format
msgid "Invalid value for %(type)s in test %(test)s on host %(host)s: %(value)r"
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/host.py:1140
#: src/vigilo/vigiconf/lib/confclasses/hosttemplate.py:130
#, python-format
msgid "Invalid group name (%s)"
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/host.py:1149
#, python-format
msgid "Invalid value for %(type)s on host %(host)s: %(value)r"
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/host.py:1199
#, python-format
msgid ""
"Host '%(host)s' inherits from both the '%(tpl1)s' and '%(tpl2)s' templates "
"(in this order), but '%(tpl1)s' already inherits from '%(tpl2)s'. The "
"templates will be reordered to satisfy dependencies."
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/host.py:1225
#, python-format
msgid "Unable to load templates for %(host)s. Possible cycle."
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/host.py:1242
#, python-format
msgid "You must associate host \"%s\" with at least one group."
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/host.py:1253
#, python-format
msgid "Can't add test %(testname)s to host %(hostname)s. Maybe a missing host class ?"
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/hosttemplate.py:312
msgid "A cycle has been detected in templates"
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/hosttemplate.py:454
#, python-format
msgid "Invalid value for %(type)s in test %(test)s in template %(tpl)s: %(value)r"
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/hosttemplate.py:495
#, python-format
msgid "Invalid value for %(type)s in template %(tpl)s: %(value)r"
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/hosttemplate.py:565
#, python-format
msgid ""
"Can't add test %(testname)s to host %(hostname)s (added by template "
"%(tplname)s). Maybe a missing host class ?"
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/test.py:333
#, python-format
msgid "Invalid entry point %(epname)s in package %(eppkg)s: %(epmodname)s"
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/test.py:384
#, python-format
msgid "Invalid hostclass \"%s\". Missing __init__.py?"
msgstr ""

#: src/vigilo/vigiconf/lib/confclasses/test.py:396
#: src/vigilo/vigiconf/lib/confclasses/test.py:429
#: src/vigilo/vigiconf/lib/confclasses/test.py:570
#, python-format
msgid "Unable to load %(file)s: %(error)s"
msgstr ""

#: src/vigilo/vigiconf/lib/dispatchator/base.py:110
msgid "Generation failed!"
msgstr ""

#: src/vigilo/vigiconf/lib/dispatchator/base.py:113
msgid "Generation successful"
msgstr ""

#: src/vigilo/vigiconf/lib/dispatchator/base.py:144
msgid "Database commit successful"
msgstr ""

#: src/vigilo/vigiconf/lib/dispatchator/base.py:148
msgid "Database commit failed"
msgstr ""

#: src/vigilo/vigiconf/lib/dispatchator/base.py:159
msgid "All servers are up-to-date. No restart needed."
msgstr ""

#: src/vigilo/vigiconf/lib/dispatchator/base.py:174
#, python-format
msgid "Current revision in the repository : %d"
msgstr ""

#: src/vigilo/vigiconf/lib/dispatchator/base.py:183
#, python-format
msgid "Cannot get revision for server: %(server)s. REASON : %(reason)s"
msgstr ""

#: src/vigilo/vigiconf/lib/dispatchator/factory.py:51
msgid ""
"You are attempting a remote deployment on the Community edition. This feature"
" is only available in the Enterprise edition. Aborting."
msgstr ""

#: src/vigilo/vigiconf/lib/dispatchator/local.py:44
msgid ""
"Vigilo server management is only available in the Enterprise edition. "
"Aborting."
msgstr ""

#: src/vigilo/vigiconf/lib/dispatchator/revisionmanager.py:71
msgid ""
"you can't go back to a former revision if you have modified your "
"configuration. Use 'svn revert' to cancel your modifications"
msgstr ""

#: src/vigilo/vigiconf/lib/dispatchator/revisionmanager.py:101
#, python-format
msgid "can't compute SVN differences for the configuration dir: %s"
msgstr ""

#: src/vigilo/vigiconf/lib/dispatchator/revisionmanager.py:128
msgid "Not updating because the 'svnrepository' configuration parameter is empty"
msgstr ""

#: src/vigilo/vigiconf/lib/dispatchator/revisionmanager.py:139
#, python-format
msgid "can't get SVN status for the configuration dir: %s"
msgstr ""

#: src/vigilo/vigiconf/lib/dispatchator/revisionmanager.py:189
#, python-format
msgid "Can't add %(path)s in repository: %(error)s"
msgstr ""

#: src/vigilo/vigiconf/lib/dispatchator/revisionmanager.py:212
#, python-format
msgid "can't remove %(path)s from repository: %(error)s"
msgstr ""

#: src/vigilo/vigiconf/lib/dispatchator/revisionmanager.py:223
msgid "Not committing because the 'svnrepository' configuration parameter is empty"
msgstr ""

#: src/vigilo/vigiconf/lib/dispatchator/revisionmanager.py:235
#, python-format
msgid "can't commit the configuration dir in SVN: %s"
msgstr ""

#: src/vigilo/vigiconf/lib/dispatchator/revisionmanager.py:240
msgid "SVN commit successful"
msgstr ""

#: src/vigilo/vigiconf/lib/dispatchator/revisionmanager.py:263
#, python-format
msgid ""
"can't execute the request to update the local copy. COMMAND %(cmd)s FAILED. "
"REASON: %(reason)s"
msgstr ""

#: src/vigilo/vigiconf/lib/dispatchator/revisionmanager.py:303
#, python-format
msgid "can't execute the request to get the current revision: %s"
msgstr ""

#: src/vigilo/vigiconf/lib/generators/base.py:86
msgid "Generators must define the generate_host() method"
msgstr ""

#: src/vigilo/vigiconf/lib/generators/file.py:98
#, python-format
msgid "No such template: %s"
msgstr ""

#: src/vigilo/vigiconf/lib/generators/manager.py:103
#: src/vigilo/vigiconf/lib/generators/manager.py:182
#: src/vigilo/vigiconf/lib/generators/manager.py:235
#, python-format
msgid "Skipping %s generator"
msgstr ""

#: src/vigilo/vigiconf/lib/generators/manager.py:106
#: src/vigilo/vigiconf/lib/generators/manager.py:239
#, python-format
msgid "Generated configuration for %s"
msgstr ""

#: src/vigilo/vigiconf/lib/generators/manager.py:140
msgid "Computing ventilation"
msgstr ""

#: src/vigilo/vigiconf/lib/generators/manager.py:153
msgid "Running generators"
msgstr ""

#: src/vigilo/vigiconf/lib/generators/manager.py:175
#, python-format
msgid "Generating configuration for %s"
msgstr ""

#: src/vigilo/vigiconf/lib/generators/manager.py:187
msgid "Running database generators"
msgstr ""

#: src/vigilo/vigiconf/lib/generators/manager.py:201
#, python-format
msgid "%(errtype)s in application %(app)s: %(error)s"
msgstr ""

#: src/vigilo/vigiconf/lib/loaders/dbloader.py:62
msgid "The 'load' method must be redefined"
msgstr ""

#: src/vigilo/vigiconf/lib/loaders/dbloader.py:116
msgid "The key attribute must be defined, or the get_key() method must be redefined"
msgstr ""

#: src/vigilo/vigiconf/lib/loaders/dbloader.py:134
#: src/vigilo/vigiconf/lib/loaders/dbloader.py:191
#, python-format
msgid "Trying to override configuration for already-defined entity \"%(entity)s\""
msgstr ""

#: src/vigilo/vigiconf/lib/loaders/xmlloader.py:92
#, python-format
msgid "XSD file does not exist: %s"
msgstr ""

#: src/vigilo/vigiconf/lib/loaders/xmlloader.py:182
#, python-format
msgid "End tag mismatch error: %(start)s/%(end)s"
msgstr ""

#: src/vigilo/vigiconf/lib/loaders/xmlloader.py:189
#, python-format
msgid "Syntax error in \"%(path)s\": %(error)s"
msgstr ""

#: src/vigilo/vigiconf/lib/loaders/xmlloader.py:364
#, python-format
msgid "Can't find an item matching %s"
msgstr ""

#: src/vigilo/vigiconf/lib/server/base.py:143
#, python-format
msgid ""
"Can't activate the configuration on %(server)s. COMMAND \"%(cmd)s\" FAILED. "
"REASON: %(reason)s"
msgstr ""

#: src/vigilo/vigiconf/lib/server/base.py:163
#, python-format
msgid "Can't tar config for server %(server)s: %(error)s"
msgstr ""

#: src/vigilo/vigiconf/lib/server/base.py:178
#, python-format
msgid "%s : deployment successful."
msgstr ""

#: src/vigilo/vigiconf/lib/server/base.py:195
#, python-format
msgid "Cannot copy files (%(from)s to %(to)s): %(error)s."
msgstr ""

#: src/vigilo/vigiconf/lib/server/base.py:258
#, python-format
msgid "Cannot write the revision file: %s"
msgstr ""

#: src/vigilo/vigiconf/lib/server/base.py:263
#, python-format
msgid ""
"Server %(server)s:\n"
"    deployed: %(deployed)d\n"
"    installed: %(installed)d\n"
"    previous: %(previous)d"
msgstr ""

#: src/vigilo/vigiconf/lib/server/base.py:276
msgid "should be deployed"
msgstr ""

#: src/vigilo/vigiconf/lib/server/base.py:278
msgid "should restart"
msgstr ""

#: src/vigilo/vigiconf/lib/server/base.py:281
msgid "disabled"
msgstr ""

#: src/vigilo/vigiconf/lib/server/base.py:286
#, python-format
msgid "Server %s cannot be disabled"
msgstr ""

#: src/vigilo/vigiconf/lib/server/base.py:288
#, python-format
msgid "Server %s cannot be enabled"
msgstr ""

#: src/vigilo/vigiconf/lib/server/factory.py:96
msgid "On the Community Edition, you can only use localhost"
msgstr ""

#: src/vigilo/vigiconf/lib/server/factory.py:110
msgid ""
"Remote server management is not possible with the Community edition. This "
"feature is only available in the Enterprise edition. Aborting."
msgstr ""

#: src/vigilo/vigiconf/lib/server/local.py:64
#, python-format
msgid "Can't deploy the tar archive for server %(server)s: %(error)s"
msgstr ""

#: src/vigilo/vigiconf/lib/server/manager.py:130
msgid "All servers are up-to-date, no deployment needed."
msgstr ""

#: src/vigilo/vigiconf/lib/server/manager.py:136
msgid ""
"The configurations files have not been transfered on every server. See above "
"for more information."
msgstr ""

#: src/vigilo/vigiconf/lib/server/manager.py:154
msgid "The revision has not been set on every server. See above for more information."
msgstr ""

#: src/vigilo/vigiconf/lib/server/manager.py:169
msgid "Switch directories was not successful on each server."
msgstr ""

#: src/vigilo/vigiconf/loaders/application.py:44
msgid "Loading applications"
msgstr ""

#: src/vigilo/vigiconf/loaders/application.py:48
msgid "Done loading applications"
msgstr ""

#: src/vigilo/vigiconf/loaders/graphgroup.py:45
msgid "Loading graph groups"
msgstr ""

#: src/vigilo/vigiconf/loaders/graphgroup.py:52
msgid "Done loading graph groups"
msgstr ""

#: src/vigilo/vigiconf/loaders/group.py:122
msgid "Loading groups"
msgstr ""

#: src/vigilo/vigiconf/loaders/group.py:126
msgid "Done loading groups"
msgstr ""

#: src/vigilo/vigiconf/loaders/host.py:87
#, python-format
msgid "Deleting leftover config file from database: %s"
msgstr ""

#: src/vigilo/vigiconf/loaders/host.py:93
msgid "Loading hosts"
msgstr ""

#: src/vigilo/vigiconf/loaders/host.py:124
#, python-format
msgid "Registering new configuration file '%(relative)s' (from %(absolute)s)"
msgstr ""

#: src/vigilo/vigiconf/loaders/host.py:151
#, python-format
msgid "Loading host %s"
msgstr ""

#: src/vigilo/vigiconf/loaders/host.py:431
msgid "Done loading hosts"
msgstr ""

#: src/vigilo/vigiconf/loaders/host.py:445
#, python-format
msgid "Unknown group \"%(group)s\" in host \"%(host)s\"."
msgstr ""

#: src/vigilo/vigiconf/loaders/host.py:474 src/vigilo/vigiconf/loaders/host.py:486
#, python-format
msgid ""
"syntax error in host %(host)s: could not find a group matching path "
"\"%(path)s\""
msgstr ""

#: src/vigilo/vigiconf/loaders/host.py:554
#, python-format
msgid ""
"warning_weight (%(warning_weight)d) must be less than or equal to weight "
"(%(weight)d) for test '%(test)s' on host '%(host)s'"
msgstr ""

#: src/vigilo/vigiconf/loaders/ventilation.py:74
msgid "Loading ventilation"
msgstr ""

#: src/vigilo/vigiconf/loaders/ventilation.py:102
#, python-format
msgid "Can't load the ventilation in database: the host %s is not in database yet"
msgstr ""

#: src/vigilo/vigiconf/loaders/ventilation.py:138
#, python-format
msgid ""
"The application %s has been removed from the configuration, it needs to be "
"stopped manually"
msgstr ""

#: src/vigilo/vigiconf/loaders/ventilation.py:149
msgid "Done loading ventilation"
msgstr ""

#: src/vigilo/vigiconf/loaders/vigiloserver.py:57
msgid "Loading Vigilo servers"
msgstr ""

#: src/vigilo/vigiconf/loaders/vigiloserver.py:71
msgid "Done loading Vigilo servers"
msgstr ""

#: src/vigilo/vigiconf/test/test_conf_testfactory.py:90
#: src/vigilo/vigiconf/test/testdata/tests/imports/Error.py:20
msgid "Import test was successful"
msgstr ""

#: src/vigilo/vigiconf/tests/all/Interface.py:91
#: src/vigilo/vigiconf/tests/all/Interface.py:99
#, python-format
msgid "Invalid value \"%(value)s\" for %(param)s. Expected one of: %(candidates)s."
msgstr ""

#: src/vigilo/vigiconf/tests/all/NTP.py:22
msgid "Test NTP has been deprecated. Please use NagiosPlugin instead"
msgstr ""

#: src/vigilo/vigiconf/tests/all/Ping.py:33
#, python-format
msgid "\"%s\" should be a list with two values"
msgstr ""

#: src/vigilo/vigiconf/tests/all/UpTime.py:40
msgid "Invalid thresholds. Expected 0 < crit < warn"
msgstr ""

